src/libDirectoryService/DirectoryService.h

```diff
@@ -535,6 +535,9 @@ class DirectoryService : public Executable, public Broadcastable {
       m_lookupStoreForGuardNodeUpdate;
   std::atomic_bool m_awaitingToSubmitNetworkInfoUpdate = {false};
 
+  bool m_doRejoinAtDSConsensus = false;
+  bool m_doRejoinAtFinalConsensus = false;
```

Comment:
Add data members as private, use function to access them

Reply:
These two variables is only for debugging purpose. Will be removed eventually, it would be a little bit redundant to do so.